.github/workflows/_charm-tests-integration.yaml

```diff
@@ -32,6 +32,12 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 1
+      - name: Setup Charmcraft's pip cache
```

Comment:
Add comment about cache details?
- lifetime
- imposed size limit
- etc

Reply:
I don't think it's necessarily relevant since the limits are **10 GB total** and **1 week**; the cache would disappear only if we don't touch a charm for a week, and even then nothing would break — it would just recreate the cache.

It's all information that can be read at the action's [repository](https://github.com/actions/cache); I don't think we should replicate information over here if it's not particularly useful